app/controllers/taxons_controller.rb

```diff
@@ -4,6 +4,8 @@ class TaxonsController < ApplicationController
   def show
     setup_content_item_and_navigation_helpers(taxon)
 
+    redirect_to(url_override, status: :temporary_redirect) and return if url_override
```

Comment:
I reckon we might want to use `.present?` on the end here, as an empty string evaluates to `true`.

Reply:
See:

```
irb(main):001:0> "" ? "bernard" : "jim"
=> "bernard"
```

Reply:
Thanks!